### PR TITLE
Slightly cleaner jq in buildscripts

### PR DESCRIPTION
**Pull request by bb010g**
I did this from the web UI; please squash & merge.

---
This change is [Reviewable](https://reviewable.io/reviews/andrebrait/adrenaline/1)

### CODE REVIEW

**Comment by andrebrait** [comment]
I totally missed this. Nice one. I'll grab the changes and test them later. Thanks.

**Comment by bb010g** [comment]
Thanks! (The commits are squashed now since it was convenient.)

File: buildscripts/docker/scripts/psp_scripts/build_pspdev.sh [hunk 1]
```diff
@@ -9,10 +9,8 @@ WORKDIR="$(pwd)"
 cd "${script_root}"
 
 latest_release_url="$(curl -L "https://api.github.com/repos/pspdev/pspdev/releases?per_page=100" \
-    | jq -r '[.[] | select(.prerelease | not)][].assets[].browser_download_url' \
-    | grep 'linux-x64' \
-    | head -n 1)"
+    | jq -r 'first(.[] | select(.prerelease | not).assets[].browser_download_url | contains("linux-x64"))')"
```
Comment (andrebrait) [inline buildscripts/docker/scripts/psp_scripts/build_pspdev.sh:12]: @bb010g FYI using `contains("linux-x64")` returned `true`.
The correct way was `select(contains("linux-x64"))` :)

Comment (bb010g) [reply buildscripts/docker/scripts/psp_scripts/build_pspdev.sh:12]: Sorry, yeah.

**Pull request merged**